near-sdk-core/src/code_generator/impl_item_method_info.rs

```diff
@@ -158,4 +158,64 @@ impl ImplItemMethodInfo {
             }
         }
     }
+
+    pub fn marshal_method(&self) -> TokenStream2 {
```

Comment:
Please add a comment to mention that it doesn't output any extra code for `wasm32` target.
Also why we can't have 2 implementations? One for `wasm32` and one for not. Does target test have to be post macro expansion?